ui/src/main/java/edu/wpi/grip/ui/util/ImageConverter.java

```diff
@@ -42,13 +42,27 @@ public Image convert(Mat mat) {
             throw new IllegalStateException("This modifies an FX object. This must be run in the UI Thread");
         }
 
+        // Copy the mat for display so we don't screw up the data
+        Mat mat = image.clone();
+
         final int width = mat.cols();
         final int height = mat.rows();
         final int channels = mat.channels();
 
         assert channels == 3 || channels == 1 :
                 "Only 3-channel BGR images or single-channel grayscale images can be converted";
 
+        // Convert the image to 8-bit for display
+        // (This does not affect the actual data image)
+        switch (mat.depth()) {
```

Comment:
Would it be better to throw an exception in a default statement, return `null`, or ignore it?